readme_template.md

```diff
@@ -386,7 +386,14 @@ like this:
 
 ```nix
 {
-  inputs.hosts.url = "github:StevenBlack/hosts";
+  inputs = {
+    nixpkgs.url = "github:NixOS/nixpkgs?ref=$YOUR-REF";
```

Comment:
This could be a little more consistent with the placeholder `<architecture>` below, but it's still an improvement over the existing README.

Reply:
`pkgs` is used for the devshell, so you can't _just_ use `lib` here. +1 to this PR as-is.